Gemfile.lock

```diff
@@ -1,34 +1,35 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.5.1)
+    better_errors (2.7.1)
       coderay (>= 1.0.0)
       erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (11.0.1)
-    coderay (1.1.2)
-    commander (4.4.7)
+    byebug (11.1.3)
+    coderay (1.1.3)
+    commander (4.5.2)
       highline (~> 2.0.0)
-    database_cleaner (1.7.0)
-    diff-lcs (1.3)
+    database_cleaner (1.8.5)
+    diff-lcs (1.4.2)
     erubi (1.9.0)
     event_sourcery (0.23.0)
     event_sourcery-postgres (0.8.0)
       event_sourcery (>= 0.14.0)
       pg
       sequel (>= 4.38)
     highline (2.0.3)
-    method_source (0.9.2)
-    mustermann (1.0.3)
+    method_source (1.0.0)
+    mustermann (1.1.1)
+      ruby2_keywords (~> 0.0.1)
     pg (0.20.0)
-    pry (0.12.2)
-      coderay (~> 1.1.0)
-      method_source (~> 0.9.0)
-    pry-byebug (3.7.0)
+    pry (0.13.1)
+      coderay (~> 1.1)
+      method_source (~> 1.0)
+    pry-byebug (3.9.0)
```

Comment:
### [pry-byebug](https://github.com/deivid-rodriguez/pry-byebug)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.7.0 → 3.9.0

[_~~change-log~~, [source-code](https://github.com/deivid-rodriguez/pry-byebug)_]





<details>
<summary>Commits</summary>

A change of **79** commits. See the full changes on [the compare page](https://github.com/deivid-rodriguez/pry-byebug/compare/v3.7.0...v3.9.0).

These are the first 10 commits:
- (6110f58) [Bump byebug from 11.0.0 to 11.0.1 (#217)](https://github.com/deivid-rodriguez/pry-byebug/commit/6110f58050e544022e829fa607a825709f59b17e)
- (0de8068) [Bump rubocop from 0.65.0 to 0.66.0 (#216)](https://github.com/deivid-rodriguez/pry-byebug/commit/0de8068c4be250c76178df0678c04e67e2e5f7d1)
- (03f8485) [Fix coverage badge](https://github.com/deivid-rodriguez/pry-byebug/commit/03f8485e89129f80fe785446c379bd7dc7ccd6ea)
- (4790fae) [Use a fixed test-reporter version](https://github.com/deivid-rodriguez/pry-byebug/commit/4790faea0fed70f0814b7e9b919f605e57df92a6)
- (29e8005) [Add missing coverage filter](https://github.com/deivid-rodriguez/pry-byebug/commit/29e8005a32a1a10bf373f18385437481e0f15281)
- (746f270) [Track test files coverage](https://github.com/deivid-rodriguez/pry-byebug/commit/746f270f05604cdf0d3886e33b90b647ea95ec55)
- (7943b79) [Merge pull request #219 from deivid-rodriguez/fix_coverage](https://github.com/deivid-rodriguez/pry-byebug/commit/7943b7914eed21773b38fde0e4ee0a245e8e3d55)
- (be555ff) [Minimize liberapay info](https://github.com/deivid-rodriguez/pry-byebug/commit/be555ff70144bcdfd430b8a18cdf808cc5506638)
- (0c29237) [Add Tidelift info under a new funding section](https://github.com/deivid-rodriguez/pry-byebug/commit/0c292376cad3f04e882c6299ac9b55b4e49c9266)
- (88499cd) [Merge pull request #220 from deivid-rodriguez/add_tidelift_i…](https://github.com/deivid-rodriguez/pry-byebug/commit/88499cdc40a20823928e76c5ffbfddf2efe323ed)


</details>